std/algorithm/package.d

```diff
@@ -17,25 +17,25 @@ $(TR $(TH Category) $(TH Submodule) $(TH Functions)
 $(TR $(TDNW Searching)
      $(TDNW $(SUBMODULE searching))
      $(TD
-        $(SUBREF searching, all) 
```

Comment:
It isn't worth the bother. (the original committer had failed to run `tolf` before committing)